gulpfile.cjs

```diff
@@ -1376,7 +1380,7 @@ function combineJavaScript(options) {
     promises.push(streamToPromise(stream));
 
     const everythingElse = ["Source/**", "!**/*.js", "!**/*.glsl"];
-    if (optimizer === "uglify2") {
+    if (minify) {
```

Comment:
I noticed while testing that this function leaves behind the empty `combineOutput` folder inside Build.

Reply:
Yeah, that's been the case forever. I wouldn't worry about it as part of this change since as you mentioned, it happens in main as well.